test/js/node/child_process/child_process.test.ts

```diff
@@ -260,6 +261,34 @@ describe("spawnSync()", () => {
     const { stdout } = spawnSync("echo", ["hello"], { encoding: "utf8" });
     expect(stdout.trim()).toBe("hello");
   });
+
+  test.if(isLinux)("should inherit for fds >= 3", async () => {
```

Comment:
The test as written works only for fds > 3 (fd 3 will be the /dev/fd dirhandle, so always open). However, as bun always opens an epoll fd, there's no risk of `fdA === 3` that I can see.